.gitignore

```diff
@@ -1,5 +1,6 @@
-*.pyc
+**.pyc
 db.sqlite3
 shops.xlsx
 *.xlsx
-shops/migrations/*py
\ No newline at end of file
+shops/migrations/*py
```

Comment:
Are you trying to exclude the migrations files? If so, note: migrations should absolutely be part of your project source files. Your source code relies on a certain database structure. The migrations make sure the database has the right structure. When you deploy your changes to a server, you always run the migrations

Reply:
Will include them from now